scene/animation/animation_mixer.cpp

```diff
@@ -868,43 +870,26 @@ bool AnimationMixer::_update_caches() {
 
-				// TODO: Currently, misc type cannot be blended. In the future,
-				// it should have a separate blend weight, just as bool is converted to 0 and 1.
+				// TODO: Currently, misc type cannot be blended.
+				// In the future, it should have a separate blend weight, just as bool is converted to 0 and 1.
 				// Then, it should provide the correct precedence value.
+				bool skip_update_mode_warning = false;
 				if (track_value->is_continuous) {
-					switch (track_value->init_value.get_type()) {
-						case Variant::NIL:
-						case Variant::STRING_NAME:
-						case Variant::NODE_PATH:
-						case Variant::RID:
-						case Variant::OBJECT:
-						case Variant::CALLABLE:
-						case Variant::SIGNAL:
-						case Variant::DICTIONARY:
-						case Variant::ARRAY:
-						case Variant::PACKED_BYTE_ARRAY:
-						case Variant::PACKED_INT32_ARRAY:
-						case Variant::PACKED_INT64_ARRAY:
-						case Variant::PACKED_FLOAT32_ARRAY:
-						case Variant::PACKED_FLOAT64_ARRAY:
-						case Variant::PACKED_STRING_ARRAY:
-						case Variant::PACKED_VECTOR2_ARRAY:
-						case Variant::PACKED_VECTOR3_ARRAY:
-						case Variant::PACKED_COLOR_ARRAY: {
-							WARN_PRINT_ONCE_ED("AnimationMixer: '" + String(E) + "', Value Track: '" + String(path) + "' uses a non-numeric type as key value with UpdateMode.UPDATE_CONTINUOUS. This will not be blended correctly, so it is forced to UpdateMode.UPDATE_DISCRETE.");
-							track_value->is_continuous = false;
-							break;
-						}
-						default: {
-						}
+					if (!Animation::is_variant_interpolatable(track_value->init_value)) {
+						WARN_PRINT_ONCE_ED("AnimationMixer: '" + String(E) + "', Value Track: '" + String(path) + "' uses a non-numeric type as key value with UpdateMode.UPDATE_CONTINUOUS. This will not be blended correctly, so it is forced to UpdateMode.UPDATE_DISCRETE.");
+						track_value->is_continuous = false;
+						skip_update_mode_warning = true;
+					}
+					if (track_value->init_value.is_string()) {
+						WARN_PRINT_ONCE_ED("AnimationMixer: '" + String(E) + "', Value Track: '" + String(path) + "' blends string, it is an experimental algorithm.");
 					}
 				}
 
-				if (was_continuous != track_value->is_continuous) {
-					WARN_PRINT_ONCE_ED("Value Track: " + String(path) + " has different update modes between some animations may be blended. Blending prioritizes UpdateMode.UPDATE_CONTINUOUS, so the process treat UpdateMode.UPDATE_DISCRETE as UpdateMode.UPDATE_CONTINUOUS with InterpolationType.INTERPOLATION_NEAREST.");
+				if (!skip_update_mode_warning && was_continuous != track_value->is_continuous) {
+					WARN_PRINT_ONCE_ED("Value Track: '" + String(path) + "' has different update modes between some animations may be blended. Blending prioritizes UpdateMode.UPDATE_CONTINUOUS, so the process treat UpdateMode.UPDATE_DISCRETE as UpdateMode.UPDATE_CONTINUOUS with InterpolationType.INTERPOLATION_NEAREST.");
```

Comment:
Language improvements:
```suggestion
					WARN_PRINT_ONCE_ED("Value Track: '" + String(path) + "' has different update modes between some animations which may be blended together. Blending prioritizes UpdateMode.UPDATE_CONTINUOUS, so the process treats UpdateMode.UPDATE_DISCRETE as UpdateMode.UPDATE_CONTINUOUS with InterpolationType.INTERPOLATION_NEAREST.");
```

I'm not sure I fully understand the warning though. Especially the "some animations may be blended" - is that a "maybe", we're not sure? Because I saw this warning in a case where the different animations were not blended.